Tracking/src/Tracking/Reco/TruthSeedProcessor.cxx

```diff
@@ -610,6 +608,23 @@ void TruthSeedProcessor::produce(framework::Event& event) {
         });
   }
 
+  // Sort tagger hits. TO DO: simplify/merge with above
+  for (std::pair<int, std::vector<int>> element : tagger_sh_count_map) {
+    std::sort(
+        element.second.begin(), element.second.end(),
```

Comment:
We can use [range-based for loops](https://en.cppreference.com/w/cpp/language/range-for) in order to make this a bit more readable. (I put an underscore in front of `track_id` to point out that it won't be used but we know what it is.)

```suggestion
  for (auto& [ _track_id, hit_indices ] : tagger_sh_count_map) {
    std::sort(
        hit_indices.begin(), hit_indices.end(),
```